ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@rakeshkumar-verma : have u checked d that all thsese default policies are needed ?

Reply:
keeping default policy for reference and will remove once develop project specific